Runtimes/Core/cmake/modules/DefaultSettings.cmake

```diff
@@ -68,6 +68,18 @@ elseif(WIN32)
   set(SwiftCore_THREADING_PACKAGE_default "WIN32")
   set(SwiftCore_ENABLE_PRESPECIALIZATION_default ON)
   set(SwiftCore_CONCURRENCY_GLOBAL_EXECUTOR_default "dispatch")
+
+  set(SwiftCore_ENABLE_COMMANDLINE_SUPPORT_default ON)
```

Comment:
Lets just enable this unconditionally: https://github.com/swiftlang/swift/pull/81491